python/README.md

```diff
@@ -0,0 +1,141 @@
+# Chericat Python
+A set of python code to display Chericat capabilities related data on graphs
+
+Packages required and their tested versions to run the programs:
+
+|Name|Tested Version|
+|---|---|
+|Courier Prime|g20180831_1|
+|Graphviz|7.1.0_7|
+|Python|3.9.17|
+|Sqlite3|3.42.0|
+
+The main program to run is "chericat_graphs.py". To show all the options available and their brief description, use the --help, -h option:
+
+```console
+$ python3.9 chericat_graphs.py --help
```

Comment:
`chericat_graphs.py` should be changed to include the shebang:
```
#!/usr/bin/env python3
```
and it should have the executable permission bit set.

Then, you can simply type:
```suggestion
$ ./chericat_graphs.py --help
```